ports/gecode/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_github(
```

```suggestion
vcpkg_fail_port_install(ON_TAREGT "windows")

vcpkg_from_github(
```
It would be better to add the unsupported message here.